src/less/awful/ssl.clj

```diff
@@ -80,16 +80,14 @@
 (defn key-store
   "Makes a keystore from a PKCS8 private key file, a public cert file, and the
   signing CA certificate."
-  [key-file cert-file ca-cert-file]
+  [key-file cert-file]
```

Comment:
So here's a really great question: Why *don't* I use the CA cert file? What was I thinking?